src/terrain/clients/apps/raw.clj

```diff
@@ -1277,8 +1277,8 @@
                    :content-type :json}))))
 
 (defn record-login
-  [ip-address user-agent]
-  (let [params (remove-nil-values {:ip-address ip-address :user-agent user-agent})]
+  [ip-address session-id login-time]
+  (let [params (remove-nil-values {:ip-address ip-address :session-id session-id :login-time login-time})]
```

Comment:
Since `/users/logout` was removed in cyverse-de/apps#277, the `record-logout` function should also be removed from this namespace.